docs/src/api.md

```diff
@@ -31,11 +31,6 @@ AgentEvent
 ```@docs
 @agent
 AbstractAgent
-@multiagent
```

Comment:
This you have to add back, right? Because you used this name now for the "agents-like" version of `@sumtype`.

Reply:
yes, thank you